src/com/lilithsthrone/game/character/GameCharacter.java

```diff
@@ -6518,6 +6506,9 @@ public String addFetish(Fetish fetish) {
 	}
 	
 	public String addFetish(Fetish fetish, boolean shortDescription) {
+		if(fetish.isContentEnabled()) {
```

Comment:
Should this be `!fetish.isContentEnabled()`?

Reply:
Yes it should be